source/draw/Color.ooc

```diff
@@ -36,7 +36,7 @@ ColorUv: cover {
 	toMonochrome: func -> ColorMonochrome { ColorMonochrome new() }
 	toYuv: func -> ColorYuv { ColorYuv new(128, this u, this v) }
 	toYuva: func -> ColorYuva { ColorYuva new(128, this u, this v, 255) }
-	toRgb: func -> ColorRgb { ColorConvert yuvToBgr(this toYuv()) toRgb() }
+	toRgb: func -> ColorRgb { ColorConvert yuvToRgb(this toYuv()) }
```

Comment:
Color conversion now goes directly between RGB and YUV so that BGR can later be removed.